lib/pronto/formatter/text_formatter.rb

```diff
@@ -1,10 +1,26 @@
 module Pronto
   module Formatter
     class TextFormatter
+      include Colorizable
+
+      LOCATION_COLOR = :cyan
+
+      LEVEL_COLORS = {
+        info: :yellow,
+        warning: :magenta,
+        error: :red,
+        fatal: :red
+      }.freeze
+
+      attr_reader :config
+
+      def initialize(config = Config.new)
```

Comment:
This is currently called here:
  * pronto/formatter/colorizable_spec.rb: [formatter_class.new(config)](https://github.com/mmozuras/pronto/pull/134/files#diff-0389d75965eaee0e29a946aa98b90797R15)
  * pronto/formatter/text_formatter_spec.rb: [described_class.new(config)](https://github.com/mmozuras/pronto/pull/134/files#diff-a8e1019a71e7c0c9aaccf75885afb994R5)

I've also tried to be consistent with other usages like:
  * pronto/lib/pronto/runners.rb: [def initialize(runners = Runner.runners, config = Config.new)](https://github.com/mmozuras/pronto/blob/master/lib/pronto/runners.rb#L3)